src/main/kotlin/org/rust/ide/statistics/RsProjectUsagesCollector.kt

```diff
@@ -110,5 +112,512 @@ class RsProjectUsagesCollector : ProjectUsagesCollector() {
             EventFields.StringList("workspace", Edition.values().map { it.presentation }),
             EventFields.StringList("dependencies", Edition.values().map { it.presentation })
         )
+
+        private val DEPENDENCY_NAME = EventFields.String("name", popularCrates)
+
+        private val DIRECT_DEPENDENCY = GROUP.registerEvent("dependency", DEPENDENCY_NAME, EventFields.Version)
     }
 }
+
+private val popularCrates = listOf(
```

Comment:
Let's describe a way how this list was generated. It may be useful if we decide to update it in the future

Reply:
I did it semi-manually so no concrete script to share unfortunately. We can come up with more streamlined version in future if needed.